appinfo/app.php

```diff
@@ -19,9 +19,12 @@
  *
  */
 
+use OCA\CustomGroups\Hooks;
+
 $app = new \OCA\CustomGroups\Application();
 $app->registerGroupBackend();
 $app->registerNotifier();
+$app->getContainer()->query(Hooks::class)->register();
```

Comment:
You should be able to call this like this: 

```suggestion
Hooks::register();
```